src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RADIUS_NUMBER = 10;
+    private final Random random = new Random();
+    private final ColorSupplier supplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        return null;
```

Comment:
Returning `null` from a method is a bad practice. Instead of returning `null`, you can implement the logic to generate and return a random `Figure` object or throw an exception if the method's purpose cannot be fulfilled.